docset/winserver2012-ps/bitlocker/Get-BitLockerVolume.md

```diff
@@ -17,55 +17,44 @@ Gets information about volumes that BitLocker can protect.
 ## SYNTAX
 
 ```
-Get-BitLockerVolume [[-MountPoint] <String[]>]
+Get-BitLockerVolume [[-MountPoint] <String[]>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 The **Get-BitLockerVolume** cmdlet gets information about volumes that BitLocker Drive Encryption can protect.
 You can specify a BitLocker volume by drive letter, followed by a colon (C:, E:).
 If you do not specify a drive letter, this cmdlet gets all volumes for the current computer.
 
-You can use this cmdlet to get BitLocker volumes to use with other cmdlets, such as the Enable-BitLocker cmdlet or the Add-BitLockerKeyProtector cmdlet.
+You can use this cmdlet to get BitLocker volumes to use with other cmdlets, such as the **Enable-BitLocker** cmdlet or the **Add-BitLockerKeyProtector** cmdlet.
 You can also use this cmdlet to view the following information about a BitLocker volume: 
 
-- VolumeType.
-Data or Operating System.
-- Mount Point.
-Drive letter.
-- CapacityGB.
-Size of drive.
-- VolumeStatus.
-Whether BitLocker currently protects some, all, or none of the data on the volume. 
-- Encryption Percentage.
-Percent of the volume protected by BitLocker.
-- KeyProtector.
-Type of key protector or protectors.
-- AutoUnlock Enabled.
-Whether BitLocker uses automatic unlocking for the volume.
-- Protection Status.
-Whether BitLocker currently uses a key protector to encrypt the volume encryption key.
-
-For an overview of BitLocker, see BitLocker Drive Encryption Overviewhttp://technet.microsoft.com/en-us/library/cc732774.aspx (http://technet.microsoft.com/en-us/library/cc732774.aspx) on TechNet.
+- VolumeType - Data or Operating System.
+- Mount Point - Drive letter.
+- CapacityGB - Size of drive.
+- VolumeStatus - Whether BitLocker currently protects some, all, or none of the data on the volume. 
+- Encryption Percentage - Percent of the volume protected by BitLocker.
+- KeyProtector - Type of key protector or protectors.
+- AutoUnlock Enabled - Whether BitLocker uses automatic unlocking for the volume.
+- Protection Status - Whether BitLocker currently uses a key protector to encrypt the volume encryption key.
+- EncryptionMethod - Indicates the encryption algorithm and key size used on the volume.
+
+For an overview of BitLocker, see [BitLocker Overview](https://docs.microsoft.com/en-us/windows/security/information-protection/bitlocker/bitlocker-overview)
+
+For an overview of Encryption methods, see [GetEncryptionMethod method](https://docs.microsoft.com/en-us/windows/win32/secprov/getencryptionmethod-win32-encryptablevolume)
```

Comment:
For an overview of encryption methods, see [GetEncryptionMethod method](https://docs.microsoft.com/windows/win32/secprov/getencryptionmethod-win32-encryptablevolume).
```